docs/Changelog.md

```diff
@@ -790,6 +790,58 @@ This version of the operator has been available since version 1 of the default O
 <dd>Constrain input and output types to float tensors.</dd>
 </dl>
 
+### <a name="Conv_Integer-1"></a>**Conv_Integer-1**</a>
+
+  The integer convolution operator consumes an input tensor, a filter and a padding value, and computes the output. The production MUST never overflow. The accumulation may overflow if and only if in 32 bits.
```

Comment:
What do you mean by `The accumulation may overflow if and only if in 32 bits.`?

Reply:
It probably means if it overflows, it only can overflow in 32 bit.

Reply:
Yes, it means it may overflow only in 32 bits, normally it should never overflow.